### DISCUSSION TITLE
FR: allow import/export of webhooks

### DISCUSSION THREAD

**Discussion by KatieMFritz** [craftcms/webhooks#54] [Ideas]
I can see why we wouldn't want to store this in Project Config, since you might want different webhooks for different environments. However, for those of us who do want to test locally and then propagate in more environments, an export/import tool would be really helpful!

**Comment by KatieMFritz** [comment]
In the meantime, using an include is helpful, and I'm putting comments in the twig file for some of the other stuff.

**Comment by weotch** [comment]
@KatieMFritz can you expand on what you meant by "using an include is helpful"? 

I'm wanting a way to store webhook configuration in a starter repo to reduce the number of steps it takes to spin up a new headless Craft instance.
